Modules/System/Azure Blob Services API/src/ABSBlobClient.Codeunit.al

```diff
@@ -600,6 +600,223 @@ codeunit 9053 "ABS Blob Client"
         exit(BlobServicesApiImpl.PutBlockFromURL(BlobName, SourceUri, BlockId, OptionalParameters));
     end;
 
+    /// <summary>
+    /// Requests a new lease. If the blob does not have an active lease, the Blob service creates a lease on the blob. The lease duration can be 15 to 60 seconds or can be infinite
+    /// see: https://docs.microsoft.com/en-us/rest/api/storageservices/lease-blob
+    /// </summary>
+    /// <param name="BlobName">The name of the blob.</param>
+    /// <param name="LeaseId">Guid containing the response value from x-ms-lease-id HttpHeader</param>
+    /// <returns>An operation reponse object</returns>
+    procedure LeaseAcquire(BlobName: Text; var LeaseId: Guid): Codeunit "ABS Operation Response"
```

Comment:
Why "<entity><verb>" and not the more usual "<verb><entity>" for function names (e. g. LeaseAcquire vs AcquireLease)?

Reply:
The only real reason was that all Lease-operations are shown alphabetically in Intellisense (like when you type "Lease" you see all operations for this. But now that I think about it: it wouldn't matter, since Intellisense is not only looking at the start of the name.
I'll adjust this to have it more coherent 👍

Reply:
Changed